DogeNet.Registration/Controllers/RegistrationController.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <copyright file="RegistrationController.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.Registration.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using DogeNet.BLL.Features.Account.CreateAccount;
+    using MediatR;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RegistrationController : ControllerBase
+    {
+        private readonly IMediator mediator;
+
+        public RegistrationController(IMediator mediator)
+        {
+            this.mediator = mediator;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Register(CreateAccountCommand command)
+        {
+            if (this.ModelState.IsValid)
```

Comment:
We need to automate validation. So each controller should not include this.ModelState.IsValid. It should be done automatically. But let me create another issue for this.

Reply:
ok